src/components/AzureMapFeature/AzureMapFeature.tsx

```diff
@@ -22,10 +25,35 @@ const AzureMapFeature = memo((props: IAzureMapFeature) => {
     }
   }, [])
 
+  useEffect(() => {
```

Comment:
My proposition here is to add optional type to azure map feature like type shape | feature. When type is a shape we create shape and make reactions if properties of coordinates are changed. So we accomplish that without implementing another context.

Reply:
https://github.com/WiredSolutions/react-azure-maps/pull/35/files

Reply:
Let's go that way! 🔥